src/PowerShellEditorServices/Session/Host/TerminalPSHostUserInterface.cs

```diff
@@ -131,7 +131,7 @@ protected override ChoicePromptHandler OnCreateChoicePromptHandler()
             ConsoleColor oldBackgroundColor = System.Console.BackgroundColor;
 
             System.Console.ForegroundColor = foregroundColor;
-            System.Console.BackgroundColor = backgroundColor;
+            System.Console.BackgroundColor = ((int)backgroundColor != -1) ? backgroundColor : oldBackgroundColor;
```

Comment:
When build PSES I can't do anything without fixing this each time otherwise PSES crashes.

Reply:
Thanks for fixing that, sorry for leaving it hanging for so long.